### ISSUE TITLE
2.0.11.4 breaks import in Google Colab examples

### ISSUE THREAD

**Issue by BptGrm** [softmatterlab/DeepTrack-2.0#86]
Running example number 4 (Multi molecule tracking) stops at the second cell when importing deeptrack : 
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-c0187a06b858> in <module>()
      2 import matplotlib.pyplot as plt
      3 
----> 4 import deeptrack as dt
      5 
      6 from deeptrack.extras import datasets

4 frames
/usr/local/lib/python3.7/dist-packages/deeptrack/models/autotracking/generators.py in <module>()
      1 from ...generators import ContinuousGenerator
      2 from ...augmentations import Affine
----> 3 from ...features import Value
      4 from .equivariances import (
      5     ScaleEquivariance,

ImportError: cannot import name 'Value' from 'deeptrack.features' (/usr/local/lib/python3.7/dist-packages/deeptrack/features.py)
```
`!pip install deeptrack` installed deeptrack 2.0.11.4. 
Explicitely stating `!pip install deeptrack==0.11.4` results in the same error.
`!pip install deeptrack==0.11.3` solves the issue.

I haven't tried replicating it on a local install yet.

EDIT :
This also seems to be the case on local install.
I created a fresh conda environment, ran `pip install deeptrack`, then ran import `deeptrack as dt`, which results in :
```
Exception has occurred: ImportError
cannot import name 'Value' from 'deeptrack.features' (C:\Users\Magnetic Tweezers\anaconda3\envs\test_dt\lib\site-packages\deeptrack\features.py)
  File "C:\Users\Magnetic Tweezers\Documents\user\isol\test.py", line 1, in <module>
    import deeptrack as dt
```
Installing deeptrack 0.11.3 also solves the issue here.

**Comment by hkabbech** [comment]
Same issue on Google colab

**Comment by BenjaminMidtvedt** [comment]
This seems to be Git not removing empty folders on checkouts, which conflicts with Python modules. Should be fixed in 0.11.5. Please reopen if the issue persists!
